core/src/main/kotlin/br/com/concrete/yosef/api/property/elementgroup/GravityCommand.kt

```diff
@@ -0,0 +1,62 @@
+package br.com.concrete.yosef.api.property.elementgroup
+
+import android.annotation.SuppressLint
+import android.view.Gravity
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import br.com.concrete.yosef.api.property.DynamicPropertyCommand
+import br.com.concrete.yosef.entity.DynamicProperty
+
+/**
+ * Command class that implements the [DynamicPropertyCommand] applying
+ * the gravity property to the view([LinearLayout])
+ *
+ * @see [FrameLayout.LayoutParams.gravity]
+ */
+class GravityCommand : DynamicPropertyCommand {
+
+    companion object {
+        /**
+         * This constant documents which name is associated with the property
+         */
+        const val GRAVITY = "gravity"
+        /**
+         * This constant represents the possible value top for the gravity property
+         */
+        const val TOP = "top"
+        /**
+         * This constant represents the possible value bottom for the gravity property
+         */
+        const val BOTTOM = "bottom"
+        /**
+         * This constant represents the possible value center for the gravity property
+         */
+        const val CENTER = "center"
+    }
+
+    @SuppressLint("RtlHardcoded")
+    override fun apply(view: View, dynamicProperty: DynamicProperty) {
+
+        if (!(dynamicProperty.value == TOP ||
+                        dynamicProperty.value == BOTTOM ||
+                        dynamicProperty.value == CENTER))
+            throw IllegalArgumentException("The value (${dynamicProperty.value}) " +
+                    "cannot be applied to property $GRAVITY.")
+
+        if (view is LinearLayout) {
```

Comment:
I think this property should be package private since it always considers a frame layout as a parent. I worry about this being used inadvertently by someone.

Reply:
Done